README.md

```diff
@@ -37,185 +41,198 @@ Simple parsing for node.js.
+You pass that representation to the nearley's tiny runtime, feed it data, and get the results.
+
+nearley uses the Earley parsing algorithm with Joop Leo's optimizations to parse complex data structures easily.
+Thanks to this algorithm, nearley can parse what other JavaScript parsers cannot.
+It can handle *any* grammar you can define in BNF.
+In fact, the nearley syntax is written in *itself* (this is called bootstrapping).
 
 PEGjs and Jison are recursive-descent based, and so they will choke on a lot of
-grammars, in particular [left recursive
-ones](http://en.wikipedia.org/wiki/Left_recursion).
+grammars, in particular [left recursive ones](http://en.wikipedia.org/wiki/Left_recursion).
 
 nearley also has capabilities to catch errors gracefully, and detect ambiguous
 grammars (grammars that can be parsed in multiple ways).
 
-Installation and Usage
-----------------------
+nearley is used by [artificial intelligence](https://github.com/ChalmersGU-AI-course/shrdlite-course-project)
+and [computational linguistics](https://wiki.eecs.yorku.ca/course_archive/2014-15/W/6339/useful_handouts)
+classes at universities, as well as [file format parsers](https://github.com/raymond-h/node-dmi),
+[markup languages](https://github.com/bobbybee/uPresent) and
+[complete programming languages](https://github.com/bobbybee/carbon).
+It's an npm [staff pick](https://www.npmjs.com/package/npm-collection-staff-picks).
 
-> **Note:** For beginners, Guillermo Webster's
-> [nearley-playground](https://omrelli.ug/nearley-playground/) is a wonderful
-> way to explore nearley interactively in your browser:
->
-> ![A screenshot of the playground](www/playground.png)
-> 
-> Enjoy!
+## Installation
 
-To use nearley, you need both a *global* and a *local* installation. The two
-types of installations are described separately below.
+nearley is published as an [NPM](https://docs.npmjs.com/getting-started/what-is-npm) package compatible with [Node.js](https://nodejs.org/en/) and browsers.
 
----
+```bash
+npm install nearley
+```
 
-To *compile* a nearley parser (a `.ne` file), you need to install the
-`nearleyc` command from npm:
+Also install the package globally if you'd like to use it directly via CLI:
 
-    $ npm install -g nearley
-    $ nearleyc parser.ne
+```bash
+npm install -g nearley
+```
 
-nearley ships with three additional tools:
-- `nearley-test` lets you quickly test a grammar against some input and see the
-  results. It also lets you explore the internal state of nearley's Earley
-  table, in case you find that interesting.
-- `nearley-unparse` inverts a parser into a generator, allowing you to create
-  random strings that match your grammar.
-- `nearley-railroad` generates pretty railroad diagrams from your parser. This
-  is mainly helpful for creating documentation, as (for example) on json.org.
+See [Using in frontend](docs/using-in-frontend.md) for instructions on how to use nearley in your browser code.
```

Comment:
Well, to clarify, using a generated parser and nearley.js is a totally reasonable and common thing to do in the browser. Instructions on doing this should be prominent.

Using the nearley _compiler_ in the browser is whacky and shouldn't be mentioned here.